wiki/Contests/en.md

```diff
@@ -2,291 +2,420 @@
+| [#10](https://osu.ppy.sh/home/news/2015-06-15-monthly-beatmapping-contest-10-ctb-only) | Hylian Lemon - Foresight Is For Losers | ![osu!catch][osu!catch] | ![][flag_CL] [ZiRoX](https://osu.ppy.sh/users/200768) | ![][flag_GB] [WildOne94](https://osu.ppy.sh/users/3482692) | ![][flag_ID] [Xinely](https://osu.ppy.sh/users/1521445) | [Results](https://osu.ppy.sh/home/news/2015-08-06-monthly-beatmap-contest-10-results-ctb) |
+| [#11](https://osu.ppy.sh/home/news/2015-08-22-monthly-beatmapping-contest-11-mania-only) | Freedom Planet - Dragon Valley (Toni Leys Remix feat. Esteban Bellucci) | ![osu!mania][osu!mania] | ![][flag_ID] [LordRaika](https://osu.ppy.sh/users/3265023) | ![][flag_VN] [Kuo Kyoka](https://osu.ppy.sh/users/2596306) | ![][flag_ID] [Shionia-Chan](https://osu.ppy.sh/users/7015183) | [Results](https://osu.ppy.sh/home/news/2015-12-03-monthly-beatmapping-contest-11-results) |
+| [#12](https://osu.ppy.sh/home/news/2016-03-04-beatmapping-contest-12-osu) | IAHN - Candy Luv | ![osu!][osu!] | ![][flag_KR] [Taeyang](https://osu.ppy.sh/users/2732340) | ![][flag_CA] [Monstrata](https://osu.ppy.sh/users/2706438) | ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) | [Results](https://osu.ppy.sh/home/news/2016-05-26-monthly-beatmapping-contest-12-results) |
+| [#13](https://osu.ppy.sh/home/news/2016-12-09-monthly-beatmapping-contest-13-now-open) | BilliumMoto - HDHR | ![osu!][osu!] | ![][flag_MY] [RyoKazuka](https://osu.ppy.sh/users/6258586) | ![][flag_ID] [Fort](https://osu.ppy.sh/users/2805457) | ![][flag_DE] [Pho](https://osu.ppy.sh/users/3624692) | [Results](https://osu.ppy.sh/home/news/2017-04-02-monthly-beatmapping-contest-13-results) |
+
+After a long hiatus, the Monthly Beatmapping Contest is being rebooted in January 2020 with a brand new schedule, ruleset, voting procedure, and reward scheme. Many aspects of the competition have been dramatically altered to make way for a more engaging and attractive contest, in which some of them are as follows:
+
+- Instead of a single song, the contest now offers a selection of several songs from a different [featured artist](/wiki/Featured_Artists) each month in which all participating mappers are free to map and submit their entries from.
+- Participating mappers now only have to submit a single difficulty instead of a complete mapset.
+- Participating mappers now have to adhere to certain beatmapping theme and/or limitation set by the contest each month, such as "Star rating may not exceed 4.00★" or "Every object should overlap another object".
```

Comment:
```suggestion
- Instead of a single song, the contest now offers a selection of several songs from a different [featured artist](/wiki/Featured_Artists) each month where all participating creators are free to map and submit their entries from.
- Participating creators now only have to submit a single difficulty instead of a complete mapset.
- Participating creators now have to adhere to certain beatmapping theme and/or limitation set by the contest each month, such as "Star rating may not exceed 4.00★" or "Every object should overlap another object".
```
tense still sounds a bit off. should probably do a more thorough check on the whole article to make sure it's consistent

also, for some reason, the [ASC wants articles to use the term "creator"](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Writing#terminology) instead of "mapper" or "beatmapper". Should probably go through and ctrl-f this

Reply:
mmmmmmmmmm really not sure about this as the term "creator" is, as far as i'm aware of, very very very very very rarely used in the mapping environment at large to refer to a mapper/beatmapper.

might get more info on this